catalog/spatial/csw/spatial-csw-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -167,7 +167,8 @@
         <argument ref="filterAdapter"/>
         <property name="schemaToTagsMapping">
             <array value-type="java.lang.String">
-                <value type="java.lang.String">http://www.opengis.net/cat/wrs/1.0=registry</value>
+                <value type="java.lang.String">http://www.opengis.net/cat/csw/2.0.2=resource</value>
```

Comment:
This was originally added so we could remove a check in the code. Since i have refactored out the store stuff and a few other things this field was able to be removed.